README.adoc

```diff
@@ -230,13 +242,18 @@ localhost. The request then asserts the host with the `validateMetric()` method.
 To execute the test cases in a particular order, put them in a `testSuite()` method.
 Label the method with the `@Test` annotation, which automatically executes when your test class runs.
 
+In addition, a few endpoint tests, `InventoryEndpointTest.java` and `SystemEndpointTest.java`, have been provided for you to
```

Comment:
If you're listing the classes, should point out the full path. Also say "are provided for you" instead of "have been provided for you"

Reply:
Fixed